packages/widget-message-meet/src/components/loading-screen/index.js

```diff
@@ -3,12 +3,15 @@ import classNames from 'classnames';
 import {FormattedMessage} from 'react-intl';
 
 import Spinner from '../spinner';
+import SparkLogo from '../spark-logo';
 import styles from './styles.css';
 
 export default function LoadingScreen() {
   return (
     <div className={classNames(`loading`, styles.loading)}>
-      <div className={classNames(`spark-logo`, styles.logo)} />
+      <div className={classNames(`spark-logo`, styles.logo)}>
+        <SparkLogo />
+      </div>
```

Comment:
This and the removal of the CSS styles looks like a regression. We can't actually have an img tag right now because of how the CDN loads files. HTML uses the HTML location to resolve the `src` field, while CSS uses the location of the CSS file.